src/components/RouteInputForm/RouteInputForm.jsx

```diff
@@ -19,13 +21,23 @@ const RouteInputForm = ({ onStartingPointChange, onDestinationChange }) => {
           ariaLabelledBy="destination-label"
         />
       </div>
+      <div>
+        <label id="arrive-by-label">Arrive By</label>
+        <DateTimeInput
+          value={arriveByValue}
+          onChange={onArriveByDateTimeChanged}
+          ariaLabelledBy="arrive-by-label"
+        />
+      </div>
     </form>
   );
 };
 
 RouteInputForm.propTypes = {
   onStartingPointChange: PropTypes.func.isRequired,
   onDestinationChange: PropTypes.func.isRequired,
+  onArriveByDateTimeChanged: PropTypes.func.isRequired,
```

Comment:
nitpick - for consistency, could we name this property `onArriveByDateTimeChange` (without the d) — see how the other change callback props are also named "on...Change" without d  
```
RouteInputForm.onStartingPointChange
RouteInputForm.onDestinationChange
DateTimeInput.onChange
```